### ISSUE TITLE
mu does not support python turtle programming

### ISSUE THREAD

**Issue by yonghuming** [mu-editor/mu#352]
* What you were trying to do,
* What steps you took to make this happen,
* What you expected to happen,
* What actually happened,
* Why this difference is problematic (it may not be a bug!),
* Technical details like the version of Mu you're using, your OS version and
  other aspects of the context in which Mu was running. 
i want to do some python turtle programming
i write a script 
```
from turtle import *
for x in range(4):
    forward(100)
    right(90)
```
if everything work i will get a square.,but actully i got an error
```
Traceback (most recent call last):
  File "c:\users\langxianmeng\mu_code\intro.py", line 1, in <module>
    from turtle import *
  File "turtle.py", line 107, in <module>
ModuleNotFoundError: No module named 'tkinter'
```

i use the ** **mu_2018-02-23_18_46_master_64db73d_32bit.exe** ** on windows 10 64 bit system
```
i guest the python version mu use does not packag the turtle
display.show(Image.HEART)
2018-02-28 14:41:49,078 - mu.logic:470(_load) INFO: Loading script from: C:/Users/langxianmeng/mu_code/main.py
2018-02-28 14:41:49,078 - mu.logic:498(_load) DEBUG: from microbit import *

while True:
    display.show(Image.HEART)
    sleep(500)
    display.show(Image.HEART_SMALL)
    sleep(500)
        
2018-02-28 14:41:49,093 - mu.logic:470(_load) INFO: Loading script from: D:/ext.py
2018-02-28 14:41:49,093 - mu.logic:498(_load) DEBUG: 
2018-02-28 14:41:49,104 - mu.logic:470(_load) INFO: Loading script from: C:/Users/langxianmeng/mu_code/intro.py
2018-02-28 14:41:49,105 - mu.logic:498(_load) DEBUG: print("hello world")
2018-02-28 14:41:49,124 - mu.logic:435(restore_session) INFO: Loaded files.
2018-02-28 14:41:49,152 - mu.logic:712(change_mode) INFO: Workspace directory: C:\Users\langxianmeng\mu_code
2018-02-28 14:41:54,466 - mu.logic:679(select_mode) INFO: Showing available modes: ['python', 'adafruit', 'microbit', 'debugger', 'pygamezero']
2018-02-28 14:41:56,772 - mu.logic:712(change_mode) INFO: Workspace directory: C:\Users\langxianmeng\mu_code
2018-02-28 14:41:59,361 - mu.modes.python3:162(run_script) DEBUG: print("hello world")
2018-02-28 14:41:59,370 - mu.interface.panes:602(start_process) INFO: Running script: c:\users\langxianmeng\mu_code\intro.py
2018-02-28 14:41:59,370 - mu.interface.panes:604(start_process) INFO: Running with interactive mode.
2018-02-28 14:41:59,370 - mu.interface.panes:607(start_process) INFO: Command args: []
2018-02-28 14:41:59,372 - mu.interface.panes:613(start_process) INFO: Working directory: C:\Users\langxianmeng\mu_code
2018-02-28 14:41:59,373 - mu.interface.panes:618(start_process) INFO: Python path: ['', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32\\python36.zip', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32\\DLLs', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32\\lib', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages', 'd:\\coding\\pgzero', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\win32', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\win32\\lib', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\Pythonwin', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\IPython\\extensions']
2018-02-28 14:42:06,763 - mu.logic:732(autosave) INFO: Autosave has detected changes.
2018-02-28 14:42:06,764 - mu.logic:736(autosave) INFO: Saving script to: C:/Users/langxianmeng/mu_code/intro.py
2018-02-28 14:42:06,764 - mu.logic:737(autosave) DEBUG: from turtle im
2018-02-28 14:42:09,251 - mu.logic:533(save) INFO: Saving script to: C:/Users/langxianmeng/mu_code/intro.py
2018-02-28 14:42:09,251 - mu.logic:534(save) DEBUG: from turtle import *
2018-02-28 14:42:10,009 - mu.modes.python3:172(stop_script) DEBUG: Stopping script.
2018-02-28 14:42:10,745 - mu.modes.python3:162(run_script) DEBUG: from turtle import *
2018-02-28 14:42:10,750 - mu.interface.panes:602(start_process) INFO: Running script: c:\users\langxianmeng\mu_code\intro.py
2018-02-28 14:42:10,751 - mu.interface.panes:604(start_process) INFO: Running with interactive mode.
2018-02-28 14:42:10,751 - mu.interface.panes:607(start_process) INFO: Command args: []
2018-02-28 14:42:10,751 - mu.interface.panes:613(start_process) INFO: Working directory: C:\Users\langxianmeng\mu_code
2018-02-28 14:42:10,751 - mu.interface.panes:618(start_process) INFO: Python path: ['', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32\\python36.zip', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32\\DLLs', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32\\lib', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages', 'd:\\coding\\pgzero', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\win32', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\win32\\lib', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\Pythonwin', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\IPython\\extensions']
2018-02-28 14:42:14,994 - mu.modes.python3:172(stop_script) DEBUG: Stopping script.
2018-02-28 14:42:16,768 - mu.logic:732(autosave) INFO: Autosave has detected changes.
2018-02-28 14:42:16,769 - mu.logic:736(autosave) INFO: Saving script to: C:/Users/langxianmeng/mu_code/intro.py
2018-02-28 14:42:16,769 - mu.logic:737(autosave) DEBUG: from turtle import *


2018-02-28 14:42:21,766 - mu.logic:732(autosave) INFO: Autosave has detected changes.
2018-02-28 14:42:21,767 - mu.logic:736(autosave) INFO: Saving script to: C:/Users/langxianmeng/mu_code/intro.py
2018-02-28 14:42:21,767 - mu.logic:737(autosave) DEBUG: from turtle import *

forw
2018-02-28 14:42:24,245 - mu.logic:533(save) INFO: Saving script to: C:/Users/langxianmeng/mu_code/intro.py
2018-02-28 14:42:24,245 - mu.logic:534(save) DEBUG: from turtle import *

forward(100)
2018-02-28 14:42:25,306 - mu.modes.python3:162(run_script) DEBUG: from turtle import *

forward(100)
2018-02-28 14:42:25,308 - mu.interface.panes:602(start_process) INFO: Running script: c:\users\langxianmeng\mu_code\intro.py
2018-02-28 14:42:25,308 - mu.interface.panes:604(start_process) INFO: Running with interactive mode.
2018-02-28 14:42:25,308 - mu.interface.panes:607(start_process) INFO: Command args: []
2018-02-28 14:42:25,309 - mu.interface.panes:613(start_process) INFO: Working directory: C:\Users\langxianmeng\mu_code
2018-02-28 14:42:25,309 - mu.interface.panes:618(start_process) INFO: Python path: ['', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32\\python36.zip', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32\\DLLs', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32\\lib', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages', 'd:\\coding\\pgzero', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\win32', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\win32\\lib', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\Pythonwin', 'C:\\Users\\langxianmeng\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\IPython\\extensions']
2018-02-28 14:42:29,506 - mu.modes.python3:172(stop_script) DEBUG: Stopping script.
2018-02-28 14:43:16,265 - mu.logic:679(select_mode) INFO: Showing available modes: ['python', 'adafruit', 'microbit', 'debugger', 'pygamezero']
2018-02-28 14:43:19,225 - mu.logic:668(show_logs) INFO: Showing logs from C:\Users\langxianmeng\AppData\Local\python\mu\Logs\mu.log
```

**Comment by martinohanlon** [comment]
In summary, Tkinter isnt installed on your computer, or Mu cant see it.

A couple of quick questions:
- are you using the windows installer? If so, have a look at https://github.com/mu-editor/mu/issues/340
- are you using linux distro? Often Tkinter isnt install by default. try `sudo apt install python3-tk` to install it.

**Comment by ntoll** [comment]
Hi @yonghuming,

This is because tkinter isn't installed with the Windows installer. This is a known problem and it's on my todo list to fix. See #340 for the ticket that's tracking this.

In summary, it's a known problem and will be fixed soon..!

**Comment by yonghuming** [comment]
@ntoll thank you i got it,

**Comment by ntoll** [comment]
Fixed by #391, try a recent Windows installer to try it out (I've tested it on Windows 7 32bit and Windows 10 64bit).
